### PR TITLE
Get literal value from parsed expression

### PR DESCRIPTION
**Pull request by anderson-joyle**
Issue https://github.com/microsoft/Power-Fx/issues/2263.

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Core/Utils/Extensions.cs [hunk 1]
```diff
@@ -163,5 +163,45 @@ internal static bool CheckAggregateNames(this DType argType, DType dataSourceTyp
...
+                    value = numLitNode.Value.Value;
+                    return true;
+                case DecLitNode decLitNode:
+                    value = decLitNode.Value.Value;
+                    return true;
+                case BoolLitNode booLitNode:
+                    value = booLitNode.Value;
+                    return true;
+                case CallNode callNode:
+                    if (callNode.ToString().Contains("GUID("))
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Utils/Extensions.cs:190]: >callNode.ToString() [](http://example.com/codeflow?start=24&length=19)

nit - don't call ToString() here. 
drill in to the identifier.

File: src/libraries/Microsoft.PowerFx.Core/Utils/Extensions.cs [hunk 2]
```diff
@@ -163,5 +163,45 @@ internal static bool CheckAggregateNames(this DType argType, DType dataSourceTyp
...
+                    {
+                        var guidStr = callNode.Args.ChildNodes[0].AsStrLit().Value;
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Utils/Extensions.cs:192]: >.AsStrLit(). [](http://example.com/codeflow?start=65&length=12)

can this return null?

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]

**Pull request merged**